<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMandatoryMarkCreator</class>
 <widget class="QDialog" name="CMandatoryMarkCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>150</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mandate label creation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="accesslevel_label">
       <property name="text">
        <string>Access level:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="MarkName_lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="accesslevel_Slider">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="Currentposition_label">
       <property name="text">
        <string>Access level : 99</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
